### PR TITLE
+ example

### PR DESCRIPTION
**Pull request by pq**
Fixes: #35.

```terminal
☁  lints [example] pub global run pana
INFO       Running `/Applications/Tools/dart-sdk/bin/dart --version`...
INFO       Running `flutter --no-version-check --version --machine`...
WARNING    pana might update or modify files in `.`.
           Analysis will begin in 15 seconds, hit CTRL+C to abort it.
           To remove this message, use `--no-warning`.
INFO       Running `/Applications/Tools/dart-sdk/bin/dart pub upgrade --verbose`...
INFO       Analyzing package...
INFO       Running `/Applications/Tools/dart-sdk/bin/dart pub get`...
INFO       Running `/Applications/Tools/dart-sdk/bin/dart pub outdated --json --up-to-date --no-dev-dependencies --no-dependency-overrides`...

## ✓ Follow Dart file conventions (20 / 20)
### [*] 10/10 points: Provide a valid `pubspec.yaml`


### [*] 5/5 points: Provide a valid `README.md`


### [*] 5/5 points: Provide a valid `CHANGELOG.md`


## ✓ Provide documentation (10 / 10)
### [*] 10/10 points: Package has an example

* Found example at: `example/README.md`

## ✓ Support multiple platforms (20 / 20)
### [*] 20/20 points: Supports 2 of 2 possible platforms (**native**, **js**)


## ✓ Pass static analysis (30 / 30)
### [*] 30/30 points: code has no errors, warnings, lints, or formatting issues


## ✓ Support up-to-date dependencies (20 / 20)
### [*] 10/10 points: All of the package dependencies are supported in the latest version

No dependencies.

To reproduce run `dart pub outdated --no-dev-dependencies --up-to-date --no-dependency-overrides`.


### [*] 10/10 points: Package supports latest stable Dart and Flutter SDKs


## ✓ Support sound null safety (20 / 20)
### [*] 20/20 points: Package and dependencies are fully migrated to null safety!


Points: 120/120.
```

/cc @devoncarew @mit-mit

### CODE REVIEW

File: example/analysis_options.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+include: package:lints/recommended.yaml
```
Comment (devoncarew) [inline example/analysis_options.yaml:1]: Include a brief 1-liner telling people where they can find more info about this file? dart.dev/go/core-lints (or other)

**Review by devoncarew — APPROVED** [review]
Do we want to include an sample dart code which gets linted? A hello world in example/lib or example/bin?

File: example/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: example
+description: An example project that uses Dart recommended lints.
+version: 0.1.0
+
```
Comment (goderbauer) [inline example/pubspec.yaml:4]: Should this specify `publish_to: none` ?

File: example/analysis_options.yaml (hunk 1 above)
Comment (pq) [reply example/analysis_options.yaml:14]: I grabbed the context from the template.

File: example/pubspec.yaml (hunk 2 above)
Comment (pq) [reply example/pubspec.yaml:5]: Done.

**Review by goderbauer — APPROVED** [review]
LGTM

**Comment by pq** [comment]
> Do we want to include and sample dart code which gets linted? 

Not sure...  If you have concrete ideas, feel free to follow-up with another PR?

**Pull request merged**